### PR TITLE
Point multibuffer function

### PR DESCRIPTION
**Pull request by osundwajeff**
Ticket #261.

Still a work in progress.

Depends on #302

### CODE REVIEW

File: geest/core/multibuffer_point.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
+from qgis.core import (
+    QgsVectorLayer,
+    QgsField,
+    QgsFeatureRequest,
+    QgsProcessingFeedback,
+    QgsCoordinateReferenceSystem,
+    QgsProcessingException,
+    edit,
+)
+from PyQt5.QtCore import QVariant
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:10]: Qt imports via the QGIS api

File: geest/core/multibuffer_point.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                "INPUT_AVOID_FEATURES": "",
+                "INPUT_AVOID_BORDERS": "",
+                "INPUT_AVOID_COUNTRIES": "",
+                "INPUT_AVOID_POLYGONS": "",
+                "OUTPUT": "memory:",
+            }
+
+            try:
+                # Run the ORS isochrones tool
+                result = processing.run(
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:83]: I guess this will go?

File: geest/core/multibuffer_point.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                    "ORS Tools:isochrones_from_layer",
+                    params,
+                    feedback=QgsProcessingFeedback(),
+                )
+                isochrone_layer = result["OUTPUT"]
+                temp_layers.append(isochrone_layer)
+
+                # Provide progress feedback
+                print(
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:92]: QgsMessageLog please

File: geest/core/multibuffer_point.py [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                    f"Processed subset {i + 1} to {min(i + self.subset_size, total_features)} of {total_features}"
+                )
+
+            except QgsProcessingException as e:
+                print(
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:97]: Same

File: geest/core/multibuffer_point.py [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+                "CRS": QgsCoordinateReferenceSystem(crs),
+                "OUTPUT": "memory:",
+            }
+            merged_result = processing.run("native:mergevectorlayers", merge_params)
+            merged_layer = merged_result["OUTPUT"]
+
+            # Proceed to create bands by differencing the ranges
+            self._create_bands(merged_layer, output_path, crs)
+        else:
+            print("No isochrones were created.")
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:115]: Print

Comment (timlinux) [inline geest/core/multibuffer_point.py:115]: Return the results?

File: geest/core/multibuffer_point.py (hunk 2 above)
Comment (osundwajeff) [reply geest/core/multibuffer_point.py:83]: Yes

File: geest/core/multibuffer_point.py [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+            "LAYERS": band_layers,
+            "CRS": QgsCoordinateReferenceSystem(crs),
+            "OUTPUT": output_path,
+        }
+        final_merge_result = processing.run(
+            "native:mergevectorlayers", merge_bands_params
+        )
+        final_layer = QgsVectorLayer(output_path, "MultiBuffer", "ogr")
+
+        print(f"Multi-buffer layer created at {output_path}")
```
Comment (timlinux) [inline geest/core/multibuffer_point.py:212]: also this one

**Comment by timlinux** [comment]
Closing @osundwajeff will make a new pr for this

**Pull request closed without merging**